### ISSUE TITLE
Make bins errors with dependency on go.uber.org/zap

### ISSUE THREAD

**Issue by mchrobak** [uber/arachne#4]
I'm unable to build the arachne binary because of dependencies on zap.

```
$ make bins
builddir build
go build -o build/arachned github.com/uber/arachne/arachned/
bootstrap.go:28:2: code in directory /home/mchrobak/.gvm/pkgsets/go1.7.5/global/src/github.com/uber-go/zap expects import "go.uber.org/zap"
make: *** [bins] Error 1
```

**Comment by Vasilis** [comment]
Hi @mchrobak , thank you for reporting this!
Zap will very soon exit the pre-1.0 phase. For now, the recommended version of Zap is pinned in the glide.yaml file.
Until I add the install_ci dependency into the makefile, you can try the following from inside the arachne directory:
```
rm -fr build/ vendor/
make install_ci
make bins
```

**Comment by mchrobak** [comment]
Thanks for the hint! I have it working now.

**Comment by Vasilis** [comment]
Resolved with commit 0e0f72e
